CMakeLists.txt

```diff
@@ -754,6 +754,23 @@ if (UNIX)
     if (nounused_avail)
       set(WARN "${WARN} -Wno-unused-but-set-variable")
     endif (nounused_avail)
+    # XXX i#3793: DynamoRIO manages '\0' termination and error states itself in
```

Comment:
3793 is this PR.  What issue do you mean to refer to?

Reply:
Fixed (i#3792).

Reply:
But I see no information on null char termination in #3792?

Reply:
The same compiler switch also turned on these warning errors in this patch. I've updated the issue. They are all somewhat related, I don't think we need a separate issue?

Reply:
Same issue seems ok so long as there is actually some information there.  Otherwise, if I were a future reader and I saw this reference I would think you got the number wrong and I would go searching the tracker looking for an issue about string termation.

Reply:
It's described in #3792 now.